tests/rare/GlobalTagThrottling.toml

```diff
@@ -0,0 +1,41 @@
+[[test]]
+testTitle='GlobalTagThrottling'
+
+  [[test.knobs]]
+  min_tag_read_pages_rate=1.0
+  global_tag_throttling=true
+
+  [[test.workload]]
+  testName='GlobalTagThrottling'
+  transactionTag='sampleTag1'
+  totalReadQuota=1.0
+
+  [[test.workload]]
+  testName='ReadWrite'
+  testDuration=600.0
+  transactionsPerSecond=100
```

Comment:
An alternative is to refactor the `WriteTagThrottling` workload to something that can also check the read throttle. Because to me, it seems the check for global tag throttling correctness is a duplicate of what `WriteTagThrottling` do.